econml/inference.py

```diff
@@ -637,8 +637,10 @@ def summary_frame(self, alpha=0.1, value=0, decimals=3, feat_name=None):
         if self.d_y == 1:
             res.index = res.index.droplevel(1)
         if self.inf_type == 'coefficient':
+            if self.fname_transformer is not None:
```

Comment:
I thought we had discussed using an identity lambda as the default for `fname_transformer` so that we'd never need to check for `None` (and I made that change to the default in several places in the last bootstrap PR, though perhaps I didn't cover them all).

Reply:
Just in case an “internal” user passes fname_transformer=None explicitly.

Reply:
I should say: the main change here is not checking that transformer is not none. Rather it is that we are always applying the fname transformer even if feat_name is none. Previously we were applying the transformer only when feat_name was not None.